packages/libraries/cli/src/helpers/errors.ts

```diff
@@ -137,6 +137,16 @@ export class MissingCdnEndpointError extends HiveCLIError {
   }
 }
 
+export class MissingTargetError extends HiveCLIError {
+  constructor() {
+    super(
+      ExitCode.ERROR,
+      errorCode(ErrorCategory.GENERIC, 9),
+      `A target is required to perform the action. Please provide the "--target" flag.`,
```

Comment:
I would recommend the `MissingArgumentsError`.
This is a generic error class that can be passed information about the target argument to better instruct users.

Reply:
good catch 🥳 changed in https://github.com/graphql-hive/console/pull/6573/commits/e84c7f387fa6414cc8f840178a1ed2755432f027